src/content/self-hosted/manage/airgap.mdx

```diff
@@ -0,0 +1,117 @@
+---
+title: AirGap Networks
+description: Installing Okteto in AirGap Networks
+sidebar_label: AirGap Networks
+id: airgap
+---
+
+import variables from '../../variables.json';
+import CodeBlock from '@theme/CodeBlock';
+import TiersList from "@theme/TiersList";
+
+# AirGap Networks<TiersList tiers="Enterprise Self-Hosted" />
+
+Okteto works on AirGap Networks, but it requires custom configuration.
+This guide has been crafted to help you overcome these hurdles, but we encourage you to [talk to us](https://www.okteto.com/get-demo/) to guide you during the installation.
+
+
+## Step 1: Disable Okteto Telemetry
+
+By default, Okteto sends [telemetry data](self-hosted/helm-configuration.mdx#telemetry) to help us understand usage patterns and prioritize improvements
+In air-gapped environments, you can disable telemetry by adding the following to your Okteto Helm configuration:
+
+```yaml
+telemetry:
+  enabled: false
+```
+
+Disabling telemetry will automatically stop Okteto CLI analytics for all developers.
+
+## Step 2: Host Images in your Private Registry
+
+By default, all the images needed to deploy the Okteto Platform, and the images used by the Okteto CLI, are hosted in DockerHub.
+In air-gapped environments, you’ll need to host these images in your private registry.
+
+### Push Images to your Private Registry
+
+For simplicity, we recommend using the same repository names and tags in your private registry.
+
+:::info
+Okteto doesn't support `imagePullSecrets`, so your cluster must be configured to have access to your Private Registry, 
```

Comment:
I don't undersand what we mean by "Okteto doesn't support imagePullSecrets". Are we talking about the helm chart?

Reply:
If so, I think we should rephrase this positively as "This guide assumes that the nodes of your cluster have image pull access to your private registry. If that's not the case, please follow your infrastructure provider's instructions on how to do it"(or something like that)